### DISCUSSION TITLE
How do I update 2.4 - abcf6e92  to 2.7?

### DISCUSSION THREAD

**Discussion by Mannshoch** [MiczFlor/RPi-Jukebox-RFID#2385] [Q&A]
apt update; apt upgrade seems not working
somewhere I read I could reuse the install script but after that the phoniebox webpage still shows 2.4

**Comment by s-martin** [comment]
apt-get update / upgrade does not work, because Phoniebox is not an apt package, but installed manually.

V2 can be upgraded via the script, _if you are running Raspberry Pi OS **Bullseye**_.
See https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Upgrade

Basically you run the installation script 
`cd; rm install-jukebox.sh; wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/master/scripts/installscripts/install-jukebox.sh; chmod +x install-jukebox.sh; ./install-jukebox.sh`
and the script reuses your existing configuration / installation.

Attention: don’t use the installation script already in your home directory, because the script itself needs to be upgraded (and also has been renamed in 2.5)

> [!Caution]
> Please backup the phoniebox folder, in case something goes wrong!
